### ISSUE TITLE
[channels 2] websocket 503 errors

### ISSUE THREAD

**Issue by brianmay** [django/channels#883]
Eventually, after a websocket is connected for several minutes, the server closes it with 503.  This is with Chrome 64.0.3282.140.

This is running the server with "./manage.py runserver" and there is only one websocket active.

However, up to the 503, the websocket appears to be working perfectly with two-way communications. No messages go missing.

The 503 message (as found with wireshark) contains the following HTML:

```
<html> <head> <title>503 Service Unavailable</title> <style>  body { font-family: sans-serif; margin: 0; padding: 0; }  h1 { padding: 0.6em 0 0.2em 20px; color: #896868; margin: 0; }  p { padding: 0 0 0.3em 20px; margin: 0; }  footer { padding: 1em 0 0.3em 20px; color: #999; font-size: 80%; font-style: italic; } </style> </head> <body> <h1>503 Service Unavailable</h1> <p>Application failed to respond within time limit.</p> <footer>Daphne</footer> </body> </html>
```

I simplified my routing.py to have:

```
application = ProtocolTypeRouter({
    "websocket": CumulocityConsumer,
})
```

I simplified the consumer to have:

```
class CumulocityConsumer(JsonWebsocketConsumer):
    """A consumer to serve Cumulocity data."""

    def connect(self):
        """Prevent adding anonymous users to the group."""
        print("2222222222222, connect")
        super().connect()

    def receive_json(self, content, **kwargs):
        """Process messages from the client to this stream."""
        print("2222222222222, receive_json", content, kwargs)
        self.send_json({
            "type": "websocket.send",
            "text": "It did or did not work.",
        })

    def disconnect(self, message):
        """Client has requested to disconnect."""
        print("2222222222222, disconnect", message)
        super().disconnect(message)
```

Even with these simplifications, I still get the same problem.

**Comment by andrewgodwin** [comment]
This bug was fixed in `daphne 2.0.2` - please try updating and reopen if it still happens on a newer version!

**Comment by brianmay** [comment]
Oh, yes, sorry your right. I missed that update.
